src/components/ha-related-items.ts

```diff
@@ -95,7 +95,7 @@ export class HaRelatedItems extends SubscribeMixin(LitElement) {
                 )}:
               </h3>
               <a
-                href="/config/integrations/config_entry/${relatedConfigEntryId}"
+                href="/config/integrations#${relatedConfigEntryId}"
```

Comment:
Let's start using query params

```suggestion
                href="/config/integrations#configEntry=${relatedConfigEntryId}"
```